core/src/main/scala/org/ensime/indexer/ClassfileIndexer.scala

```diff
@@ -66,8 +66,8 @@ final class ClassfileIndexer(file: FileObject) extends SLF4JLogging {
       val interfaceNames: List[ClassName] = interfaces.map(ClassName.fromInternal)(collection.breakOut)
       val superClass = Option(superName).map(ClassName.fromInternal)
 
-      internalRefs.addAll(interfaceNames.asJava)
-      internalRefs.addAll(superClass.toList.asJava)
+      internalRefs.addAll(interfaceNames.map(FullyQualifiedReference(_, None)).asJava)
```

Comment:
maybe `interfaceNames` should always have been a Java collection in the first place, to avoid the conversion.